pom.xml

```diff
@@ -333,6 +347,7 @@
                          exclude integration tests here. -->
                     <excludes>
                         <exclude>it/**/*.java</exclude>
+                        <exclude>upgrade/**/*.java</exclude>
```

Comment:
this exclusion can now be removed, as it is part of the property on the line below, I'll update